tests/unit/test.program.js

```diff
@@ -15,10 +15,10 @@ import {ConsoleStream} from '../../src/util/logger';
 
 describe('program.Program', () => {
 
-  function run(program, options = {}) {
+  function execWebExt(program, options = {}) {
```

Comment:
I realize now that what this function does is execute a program object (not necessarily the "web-ext" program). Therefore, I think it would make more sense to call it `execProgram()`.